src/test/java/spoon/test/model/SwitchCaseTest.java

```diff
@@ -365,4 +367,52 @@ public void test_addCaseAt_throwsIndexOutOfBoundsException_whenPositionIsOutOfBo
 			assertThrows(IndexOutOfBoundsException.class, () -> switchExpression.addCaseAt(3, onlyCase));
 		}
 	}
+	
+	@DisplayName("should return the first case expression")
+	@Test
+	public void test_getCaseExpression_shouldReturnTheFirstCaseExpression() {
+		Factory factory = new Launcher().getFactory();
+
+		CtCase<Integer> ctCase = factory.createCase();
+		CtExpression<Integer> first = factory.createCodeSnippetExpression("1");
+		CtExpression<Integer> second = factory.createCodeSnippetExpression("2");
+		CtExpression<Integer> third = factory.createCodeSnippetExpression("3");
```

Comment:
Either compile the expressions or create literals. I'd do the latter.

```suggestion
		CtExpression<Integer> first = factory.createLiteral(1);
		CtExpression<Integer> second = factory.createLiteral(2);
		CtExpression<Integer> third = factory.createLiteral(3);
```

In general you should avoid having raw code snippet expressions in a tested AST as they aren't type checked.